po/common/common.pot

```diff
@@ -7059,3 +7059,23 @@ msgstr "Last login date"
 msgctxt "last_import"
 msgid "Last import date"
 msgstr "Last import date"
+
+msgctxt "number_of_products_without_nutriscore"
+msgid "Number of products without nutri-score"
```

Comment:
Coud you capitalize the translations in this way: Nutri-Score ? That's how we put it everywhere

Reply:
Alright